app/src/main/java/com/mobiledevelopmentfoodapp/dto/Food.kt

```diff
@@ -4,7 +4,10 @@ package com.mobiledevelopmentfoodapp.dto
 //import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Food(var name: String, var description: String, var price: Double, @SerializedName("id") var productId:Int = 0 ) {
+data class Food(var name: String,
+                var description: String,
+                var price: Double,
+                @SerializedName("id") var productId:Int = 0, ) {
```

Comment:
Changed the formatting of this class header according to https://kotlinlang.org/docs/coding-conventions.html#class-headers.

Reply:
Added in trailing commas to simplify code generation according to https://kotlinlang.org/docs/coding-conventions.html#trailing-commas.